src/v3/src/components/Widget/index.tsx

```diff
@@ -286,8 +286,7 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
     >
       {/* Note that we need two theme providers until we fully migrate to odyssey-mui */}
       <MuiThemeProvider theme={mapMuiThemeFromBrand(brandColors)}>
-        {/* the style is to allow the widget to inherit the parent's bg color */}
-        <ScopedCssBaseline sx={{ backgroundColor: 'inherit' }}>
```

Comment:
looks like there was some conflict that resolved the wrong way after this was merged in the other PR

Reply:
Arrgh, git thinks its smarter than me 😅 

TY!